x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -503,43 +503,62 @@ export const getEndpointConsoleCommands = ({
 
   switch (agentType) {
     case 'sentinel_one':
-      return adjustCommandsForSentinelOne({ commandList: consoleCommands });
+      return adjustCommandsForVendor({
+        commandList: consoleCommands,
+        vendor: 'sentinel_one',
+        featureFlags: {
+          isHostIsolationEnabled: featureFlags.responseActionsSentinelOneV1Enabled,
+          isGetFileFeatureEnabled: featureFlags.responseActionsSentinelOneGetFileEnabled,
+        },
+      });
+    case 'crowdstrike':
+      return adjustCommandsForVendor({
+        commandList: consoleCommands,
+        vendor: 'crowdstrike',
+        featureFlags: {
+          isHostIsolationEnabled: featureFlags.responseActionsCrowdstrikeManualHostIsolationEnabled,
+        },
+      });
     default:
       // agentType === endpoint: just returns the defined command list
       return consoleCommands;
   }
 };
 
 /** @private */
-const adjustCommandsForSentinelOne = ({
+const adjustCommandsForVendor = ({
```

Comment:
FYI - I might revert this back to being S1 specific. and creating a function that handles only CrowdStrike in the near future. Your assumption here is that all commands are defined teh same, but we already know that `kill-process` for S1 will have its arguments adjusted to support process name instead of `pid`. My assumption is that we'll likely hit those cases with other commands as well and personally I think it would be simpler to have agentType specific adjustment functions instead of a  big one that has several code branches to handle differences between vendors

Reply:
Thanks!  I'll revert it myself 👍